python/MET_Setup_cff.py

```diff
@@ -100,6 +93,16 @@ def setup_MET(process, cms, options, postfix="PFlow"):
     # process.shiftedPatJetEnUpNoHF.jetCorrLabelUpToL3Res = cms.InputTag("ak4PFCHSL1FastL2L3Corrector")
 
 
+    # recalculate MET without HF
```

Comment:
yes but at the time I tried to make sure it was backwards compatible so older files could run over it - i can remove